forge-gui-android/pom.xml

```diff
@@ -204,18 +204,17 @@
                                 <version>6.2.2</version>
                             </dependency>
                         </dependencies>
-                        <version>4.6.1</version>
+                        <version>4.6.0</version>
                         <extensions>true</extensions>
                         <configuration>
                             <sign>
                                 <debug>true</debug>
                             </sign>
                             <sdk>
                                 <platform>26</platform>
-                                <buildTools>30.0.3</buildTools>
+                                <buildTools>34.0.0</buildTools>
```

Comment:
This might be problematic with android build?

Reply:
https://github.com/Card-Forge/forge/blob/master/.github/workflows/snapshots-android.yml#L50

Reply:
Yeah I need to update this too


I wonder why github doesn't have a problem with d8, locally I needed to fix something, or was that done with older maven version?

Reply:
Yea i don't remember if it was an issue or not, I know Agetian said thats the version he uses specifically and i had some problems doing a bunch of the different pieces, i don't know if this was one of them. If the snapshot works fine with it. I'm cool with updating. But maybe we also want to separate those changes to reduce variables.